### ISSUE TITLE
aggregate returns different field names

### ISSUE THREAD

**Issue by wshager** [persvr/rql#62]
See #61 
The test fails because the field is renamed to '0'. This part of the implementation seems to be undocumented.

**Comment by wshager** [comment]
I updated the test to reflect the undocumented feature. I'll leave this open because of the omission the specification.

**Comment by wshager** [comment]
See #63

**Comment by neonstalwart** [comment]
@wshager it seems that the `'0'` corresponds to the index of the aggregation function.  this makes it possible to do multiple aggregations on the same data and extract each result independently.

**Comment by wshager** [comment]
I assume Kris will have to update the specs? Hopefully he can find a minute to do that.

**Comment by neonstalwart** [comment]
yeah, you're probably not happy to hear this, but personally i'm not interested in the specs so they won't get any of my time - the code is all the spec i have need for right now.  my main concern is keeping this codebase stable.  my limited energy will only be invested towards that for now.

**Comment by wshager** [comment]
@neonstalwart thanks no problem
